packages/cli-config/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "../../tsconfig.base.json",
+
+  "compilerOptions": {
+    "outDir": "./build",
+    "rootDir": "./src"
+  },
+  "include": ["src"],
+  "references": [
+    { "path": "../config/tsconfig.json" },
+    { "path": "../geo/tsconfig.json" },
+    { "path": "../server/tsconfig.json" },
```

Comment:
Do we still need `server, lambda-tiler, and tiler-sharp` here? And also we are missing `config-loader`